robot.py

```diff
@@ -32,6 +35,13 @@ def robotInit(self) -> None:
         # Instantiate our RobotContainer.  This will perform all our button
         # bindings, and put our autonomous chooser on the dashboard.
         self.container = robotcontainer.RobotContainer()
+        self.container.swerve.reset_modules()         
+        """
```

Comment:
This should be a comment, not a string.  Strings aren't comments -- they're things that you print or put in a window or write to a file or something like that.